GPU/D3D11/GPU_D3D11.cpp

```diff
@@ -146,11 +146,11 @@ void GPU_D3D11::CheckGPUFeatures() {
 		features |= GPU_SUPPORTS_DUALSOURCE_BLEND;
 	if (draw_->GetDeviceCaps().depthClampSupported)
 		features |= GPU_SUPPORTS_DEPTH_CLAMP;
-	features |= GPU_SUPPORTS_ANY_COPY_IMAGE;
+	features |= GPU_SUPPORTS_COPY_IMAGE;
 	features |= GPU_SUPPORTS_TEXTURE_FLOAT;
 	features |= GPU_SUPPORTS_INSTANCE_RENDERING;
 	features |= GPU_SUPPORTS_TEXTURE_LOD_CONTROL;
-	features |= GPU_SUPPORTS_FBO;
+	features |= GPU_SUPPORTS_FRAMEBUFFER_BLIT;
```

Comment:
isn't `BlitFramebuffer` stubbed for `d3d11` ?
https://github.com/hrydgard/ppsspp/blob/master/ext/native/thin3d/thin3d_d3d11.cpp#L1414-L1419

Reply:
Right you are, oops.